saleor/graphql/core/tests/test_pagination.py

```diff
@@ -239,6 +240,17 @@ def test_pagination_invalid_cursor(books):
     assert str(result.errors[0]) == "Received cursor is invalid."
 
 
+def test_pagination_invalid_cursor_and_valid_base64(books):
+    cursor = base64.b64encode(str.encode(f"{['Test']}")).decode("utf-8")
```

Comment:
Could we add a docstring like:
`This cursor should be int value but we pass string value`